api-reference/50 Common/utils/ui/dxScheduler/getTimeZones(date).md

```diff
@@ -5,13 +5,38 @@ export: dxScheduler.getTimeZones
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
-
-##### return: Array<Object>
-<!-- Description goes here -->
+Gets the list of time zones.
 
 ##### param(date): Date | undefined
-<!-- Description goes here -->
+A date for which to get time zones. If undefined, the current date is used.
+
+##### return: Array<Object>
+A list of time zones.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Each time zone is an object with the following fields:
+
+* **id**: String
+A time zone text string from the IANA database.
+
+* **offset**: Number
+A GMT offset.
+
+* **title**: String
+A formatted time zone representation. Includes the **id** and **offset** values.
+
+`getTimeZones` accepts a past date to get a list of time zones valid at a particular moment. This is because time zones change due to daylight savings or other reasons.
```

Comment:
Let's move this paragraph to before the list.

```suggestion
This method can accept a past date to get a list of time zones valid at a particular moment. This is because time zone offsets can change, for example, due to daylight savings.
```